tasks/main.yml

```diff
@@ -50,6 +50,17 @@
   tags:
   - install
 
+- name: install the required dependencies (MacOS)
+  homebrew:
+    name:
+      - openssl@1.1
+      - python3
```

Comment:
Please fix the indentation so that it matches the style in the rest of the role.
```suggestion
    - openssl@1.1
    - python3
```

Reply:
Fixed